src/main/java/edu/kis/vh/nursery/defaultCountingOutRhymer.java

```diff
@@ -2,17 +2,21 @@
 
 public class defaultCountingOutRhymer {
 
-	private int[] NUMBERS = new int[12];
+	private static final int CONST_MINUS1 = -1;
 
-	public int total = -1;
+	private static final int CONST_12 = 12;
```

Comment:
Change this variable name same as above. Don't use magic numbers. When you are giving the name of a variable, describe what it does, not what value it stores